MultiplayerMod/Game/Chores/ChoreConsumerEvents.cs

```diff
@@ -0,0 +1,56 @@
+using System;
```

Comment:
I've checked, it will be painful and quite unnatural to make structure changing pre-commits for proper git handling. Also it's mostly because of auto-format, so I think this is just ok at this stage.